processors/baggage/processor.go

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package baggage
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/attribute"
+	otelbaggage "go.opentelemetry.io/otel/baggage"
+	"go.opentelemetry.io/otel/sdk/trace"
+)
+
+type baggageSpanProcessor struct{}
```

Comment:
Can we export the type so that we can possibly add additional methods later if needed?

Reply:
@pellared we could, though I'm not sure if there's a need just yet. I don't have a strong preference one way or the other, happy to follow your guidance on this.

Reply:
> though I'm not sure if there's a need just yet

It is in order to avoid making a breaking change.

Lets rename it to


```suggestion
type SpanProcessor struct{}
```

Reply:
👍🏻

Reply:
Renamed and made public.